src/bundle/Resources/views/admin/policy/list.html.twig

```diff
@@ -25,9 +25,9 @@
                 <td>{{ policy.function }}</td>
                 <td>
                     {% for limitation in policy.limitations %}
-                        {{ limitation.identifier }}
-
-                        ({{ limitation.limitationValues | join(',') }})
+                        <strong>{{ limitation.identifier }}</strong>:
+                        {{ ez_render_limitation_value(limitation) }}
+                        {% if not loop.last %}<br />{% endif %}
```

Comment:
`<br>` it to break line of solid text, in this case, I believe each line is own statement. If we look at this as a list of limitation we can do something like: 
```html
<ul>
    <li><span>{{ limitation.identifier }}</span>: {{ ez_render_limitation_value(limitation) }}</li>
    <li><span>{{ limitation.identifier }}</span>: {{ ez_render_limitation_value(limitation) }}</li>
</ul>
```

Some CSS classes might be needed to style.